WordPressKit/ActivityServiceRemote.swift

```diff
@@ -148,5 +180,22 @@ private extension ActivityServiceRemote {
 
         return (activities, totalItems)
     }
+    
+    func mapActivityGroupsResponse(_ response: AnyObject) throws -> ([ActivityGroup]) {
+        
+        guard let json = response as? [String: AnyObject],
+              let rawGroups = json["groups"] as? [String: AnyObject] else {
+                throw ActivityServiceRemote.ResponseError.decodingFailure
+        }
+        
+        let groups: [ActivityGroup] = try rawGroups.map { (_, value) -> ActivityGroup in
```

Comment:
Looking at Calypso, it seems like the `Activity Type` filters are displaying `ActivityGroup.name` (_e.g. Posts and Pages, Media_) so I think we can ignore the dynamic key names in the `groups` dict (_e.g. post, attachment_).
```
{
  "groups": {
    "post": {
      "name": "Posts and Pages",
      "count": 10
    },
    "attachment": {
      "name": "Media",
      "count": 5
    },
    ...
  }
}
```
![Screen Shot 2020-11-27 at 17 06 42](https://user-images.githubusercontent.com/6711616/100425906-63786780-30d3-11eb-9d1d-d46386635b00.png)

Reply:
I added a comment about the key just above. :)